src/main/java/no/nav/tag/sykefravarsstatistikk/api/provisjonering/importering/StatistikkImportController.java

```diff
@@ -0,0 +1,37 @@
+package no.nav.tag.sykefravarsstatistikk.api.provisjonering.importering;
+
+import lombok.extern.slf4j.Slf4j;
+import no.nav.security.oidc.api.Protected;
+import no.nav.tag.sykefravarsstatistikk.api.domene.statistikk.SykefraværsstatistikkLand;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Profile;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@Profile({"local", "dev"})
+@Protected
+@RestController
+@Slf4j
+@RequestMapping(value = "provisjonering/import")
+public class StatistikkImportController {
+
+    private final StatistikkImportService service;
+
+    @Autowired
+    public StatistikkImportController(StatistikkImportService service){
+        this.service = service;
+    }
+
+    @GetMapping(value = "/verif/datavarehus/landstatistikk")
```

Comment:
Jeg er litt usikker på om vi burde ha et endepunkt som bare blir brukt for verifikasjon av oss :/  Eller er det tenkt at den skal fjernes etter vi har importert ting?

Reply:
Strek den, jeg så TODO-en i servicen nå :)